general/cisetup/cisetup.go

```diff
@@ -457,58 +450,67 @@ func (cc *CiSetupCommand) getPiplinesCompletionInstruction(pipelinesFileName str
 		getPipelineUiPath(serviceDetails.Url, pipelinesFileName), ""}, nil
 }
 
-func (cc *CiSetupCommand) getJenkinsCompletionInstruction(jenkinsFileName string) ([]string, error) {
-
+func (cc *CiSetupCommand) getJenkinsCompletionInstruction(jenkinsFileName string) []string {
 	JenkinsCompletionInstruction := []string{"", colorTitle("Completing the setup"),
-		"We configured the JFrog Platform and generated a Jenkinsfile file for you under" + cc.data.LocalDirPath,
+		"We configured the JFrog Platform and generated a Jenkinsfile file for you under " + cc.data.LocalDirPath,
 		"To complete the setup, follow these steps:",
-		" Open the Jenkinsfile for edit."}
+		"* Open the Jenkinsfile for edit."}
 	// M2_HOME instructions relevant only for Maven
-	if cc.data.DetectedTechnologies[cisetup.Maven] {
+	if cc.data.BuiltTechnologies[cisetup.Maven] != nil {
```

Comment:
I think it will be better to return the following line (483) that I removed by mistake yesterday:
<img width="1044" alt="Screen Shot 2021-05-21 at 6 36 59" src="https://user-images.githubusercontent.com/24526180/119078534-1086c980-b9ff-11eb-8cf0-d563b6b8471d.png">

Reply:
@talarian1 - I think that returning this single line alone can cause issues, because of the return statements there...
Since the existing code is working, I suggest that you improve this in a follow-up PR.